spec/controllers/v0/profile/ch33_bank_accounts_controller_spec.rb

```diff
@@ -61,6 +61,31 @@ def expect_find_ch33_dd_eft_res
     end
   end
 
+  describe '#payload_session' do
+    let(:user) { create(:user) }
+    let(:session_created_at) { Time.current.utc - 1.hour }
+    let(:sso_logging_info) { { user_uuid: nil, sso_cookie_contents: nil, request_host: 'www.example.com' } }
+    let(:expected_log_message) { 'Ch33BankAccountsController#user transaction session completed:' }
+    let(:sec) { 'sec' }
+
+    before do
+      allow(controller).to receive(:current_user).and_return(user)
+      session[:created_at] = session_created_at
+    end
+
+    it 'logs the user in transaction completed' do
+      expected_session_data = {
+        user_id: user.identity,
+        completed_session: (Time.current.utc - session_created_at).to_i
+      }
+
+      expect(Rails.logger).to receive(:warn)
+        .with(expected_log_message + expected_session_data.to_s + sec, sso_logging_info)
+
+      controller.payload_session
+    end
+  end
+
   describe '#update' do
```

Comment:
You'll want to add the specs for this inside the `#update` function, since the behavior that's changing is a behavior that occurs when the `update` route is called on this controller.